model/src/main/java/de/danoeh/antennapod/model/feed/VolumeAdaptionSetting.java

```diff
@@ -32,4 +37,31 @@ public int toInteger() {
     public float getAdaptionFactor() {
         return adaptionFactor;
     }
+
+    @Nullable
+    private static Boolean boostSupported = null;
+
+    @Nullable
+    public static boolean isBoostSupported() {
+        if (boostSupported == null) {
+            boostSupported = false;
+
+            final AudioEffect.Descriptor[] audioEffects = AudioEffect.queryEffects();
+            if (audioEffects != null) {
+                for (AudioEffect.Descriptor effect : audioEffects) {
+                    if (effect.type.equals(AudioEffect.EFFECT_TYPE_LOUDNESS_ENHANCER)) {
+                        boostSupported = true;
+                        break;
+                    }
+                }
+            }
+        }
+
+        return boostSupported;
+    }
```

Comment:
Why is this nullable?

You can simplify the code a bit (and reduce the indention depth) by turning around the `if` statement:

```suggestion
    @Nullable
    public static boolean isBoostSupported() {
        if (boostSupported != null) {
            return boostSupported;
        }
        final AudioEffect.Descriptor[] audioEffects = AudioEffect.queryEffects();
        if (audioEffects != null) {
            for (AudioEffect.Descriptor effect : audioEffects) {
                if (effect.type.equals(AudioEffect.EFFECT_TYPE_LOUDNESS_ENHANCER)) {
                    boostSupported = true;
                    return boostSupported;
                }
            }
        }
        boostSupported = false;
        return boostSupported;
    }
```